### ISSUE TITLE
After build 491, //deform does not work

### ISSUE THREAD

**Issue by braceyourself07** [boy0001/FastAsyncWorldedit#442]
No error in console, all seems fine.  However, when using the //deform command with seemingly any formula the result is not consistent with what should be output... sometimes the output clears the entire area affected.

FAWE Debugpaste Link:  https://hastebin.com/epepehazoh

Before:  http://prntscr.com/edey60
Applied //deform x/=2;y/=2;z/=2 (which should grow the block from 1x1 to 2x2)
After:  http://prntscr.com/edeye1 (the entire area is just filled with the block)

Before: http://prntscr.com/edeyr9
Applied //deform rotate(x,z,90)
After:  http://prntscr.com/edeyxm

**Checklist**:

- [x] I included a `/fawe debugpaste` link
- [x] I made sure there aren't duplicates of this report [(Use Search)](https://github.com/boy0001/FastAsyncWorldedit/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of FAWE
- [x] I Made sure the bug/error is not caused by any other plugin

**Comment by dordsor21** [comment]
finally, someone stopped complaining and took 5 mins to submit a report. Thank you @braceyourself07

**Comment by boy0001** [comment]
It was actually caused by this build, http://ci.athion.net/job/FastAsyncWorldEdit/498
